.github/workflows/refapp-2x-patient-encounter.yml

```diff
@@ -0,0 +1,42 @@
+name: RefApp 2.x Patient Encounter
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Add path here to segregate the test from running against the 3.x tests. Check the existing yml file for 2.x test to borrow a leaf!